app/assess/templates/landing.html

```diff
@@ -6,7 +6,7 @@
 {% block content %}
 <p class="govuk-heading-xl">Assessor dashboard</p>
 <h1 class="govuk-heading-l">All assessments</h1>
-<h2 class="govuk-heading-m dashboard-deadline ">Due date: <span class="govuk-body-l">{{ assessment_deadline|datetime_format("%d/%m/%Y at %H:%M") }}</span></h2>
+<h2 class="govuk-heading-m dashboard-deadline ">Due date: <span class="govuk-body-l">{{ assessment_deadline|datetime_format("%d %B %Y at %H:%M") }}</span></h2>
```

Comment:
This was accidentally changed on another pr, this should now match the prototype